src/appendices/installing-software/setting-up-vsc.rst

```diff
@@ -0,0 +1,45 @@
+.. _vsc-install:
+
+Setting Up Visual Studio Code
+=============================
+
+.. index:: ! IDE, ! integrated development environment
+
+Before we start coding on our computer, we need to make sure we have the right tools! Programmers use **integrated development environments** (IDE) to write and run their code.
+The development environment we will be using for this class is Visual Studio Code.
+In addition to simply writing and running code, Visual Studio Code has tools that recognize errors in our code and has an integrated terminal so we can navigate through our filesystem to find the files that need our attention.
+
+Mac Users
+---------
+
+Go to the Visual Studio Code `download page <https://code.visualstudio.com/download/>`_ and download Visual Studio Code for Mac.
+
+.. note::
```

Comment:
How about moving this above the Mac section so that all users can read it?

Reply:
Reworked this!